scripts/us_fed/treasury_constant_maturity_rates/generate_csv_and_mcf.py

```diff
@@ -131,10 +130,8 @@ def generate_mcf():
             index += 1
 
 
-def main(argv):
-    # unused
-    del argv
-
+def main(_):
+    """Run the code."""
```

Comment:
> The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable.""")

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
Fixed.